ml-agents/mlagents/trainers/stats.py

```diff
@@ -58,6 +62,45 @@ def write_text(self, category: str, text: str, step: int) -> None:
         pass
 
 
+class ConsoleWriter(StatsWriter):
+    def __init__(self):
+        self.training_start_time = time.time()
+
+    def write_stats(
+        self, category: str, values: Dict[str, StatsSummary], step: int
+    ) -> None:
+        is_training = "Not Training."
+        if "Is Training" in values:
+            stats_summary = stats_summary = values["Is Training"]
+            if stats_summary.mean > 0.0:
+                is_training = "Training."
+        if "Environment/Cumulative Reward" in values:
+            stats_summary = values["Environment/Cumulative Reward"]
+            logger.info(
+                "{}: Step: {}. "
+                "Time Elapsed: {:0.3f} s "
+                "Mean "
+                "Reward: {:0.3f}"
+                ". Std of Reward: {:0.3f}. {}".format(
+                    category,
```

Comment:
so it looks like category is replacing run_id. is the run_id available somewhere in the logs? can you also explain the motivation behind replacing run_id by category?

Reply:
`run_id` isn't part of the StatsWriter abstraction (but can be added if there's enough demand). By default the trainer's `category` is the `run_id + brain_name`, so the same info is available to the user. Also, the `category` matches what's written to TensorBoard and the filename of the CSV, so it's consistent.

Reply:
👍